### ISSUE TITLE
Inspect errors after adding multiple video embeds

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#12284]
**Describe the bug**
I noticed that in the Inspect there are now 6 errors in relation to the 6 video embeds that I added.

tbvws-bulk-script.js:1 Uncaught DOMException: Blocked a frame with origin "https://www.youtube.com" from accessing a cross-origin frame.

Uncaught DOMException: Blocked a frame with origin "https://www.youtube.com" from accessing a cross-origin frame.
    at o (file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:5115)
    at Object.375 (file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:5262)
    at t (file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:110)
    at Object.374 (file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:4560)
    at t (file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:110)
    at file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:451
    at file:///Users/paaljoachimromdahl/Library/Application%20Support/Google/Chrome/Default/Extensions/lmjnegcaeklhafolokijcfjliaokphfk/7.3.5.0_0/content/tbvws-bulk-script.js:1:462

![screen shot 2018-11-25 at 10 34 44](https://user-images.githubusercontent.com/5323259/48977606-cfb3d100-f09d-11e8-8b41-911a31aaf0d9.png)

**To Reproduce**
1. (Not sure) I added youtube video embeds. Worked on aligning them and noticed a sloweness in the auto save. I then checked Inspect and noticed the error.

**Expected behavior**
No errors.

**Desktop (please complete the following information):**
 - OS: Max OSX 10.13.6
 - Browser Chrome and Desktop Server
 - Gutenberg Version 4.5.1

**Comment by paaljoachim** [comment]
Btw here are some pages where one can also notice the errors:
https://arnenaessproject.org/arne-naess/
https://arnenaessproject.org/eradicating-ecocide/
https://arnenaessproject.org/the-arne-naess-projects/sustainable-consumption/

I switched themes just to see if that would make a change but the error was stil there. I also turned off all plugins except Gutenberg Version 4.5.1.

If the above pages do not show an js Inspect error when anyone else checks it then it might have to do with the location I am at (I out in the wilderness so to speak taking care of 7 Greenland dogs and have satellite broadband) and not Gutenberg. I will be back home in Oslo in a few days and will check again then.
